### PR TITLE
docs(proxy-state-tree): extend error message to mention another potential root cause

### PR DESCRIPTION
**Pull request by zeusdeux**
As per [our twitter conversation](https://twitter.com/muditameta/status/1184882402897911810?s=21)

### CODE REVIEW

**Comment by zeusdeux** [comment]
Well that's a weird red build. I don't see `vue` at all in the package-lock on this branch _or_ master 🤔

**Comment by zeusdeux** [comment]
Oh wait a sec, vue@2.5.16 is a devDependency whereas vue@2.6.10 is a dependency.
Should I align the two as a part of this PR or is it disparate for a reason?

**Comment by zeusdeux** [comment]
@christianalfoni would you mind taking a look please? Thank you! ^_^

File: packages/node_modules/proxy-state-tree/src/Proxyfier.ts [hunk 1]
```diff
@@ -78,9 +78,10 @@ export class Proxifier {
     if (this.tree.master.options.devmode && !this.tree.canMutate()) {
       throw new Error(
         `proxy-state-tree - You are mutating the path "${path}", but it is not allowed. The following could have happened:
-        
+
         - You are passing state to a 3rd party tool trying to manipulate the state
         - You are running asynchronous code and forgot to "await" its execution
+        - You are attempting to mutate state outside of an action
```
Comment (christianalfoni) [inline packages/node_modules/proxy-state-tree/src/Proxyfier.ts]: This is awesome, though `proxy-state-tree` does not really have actions. So if you could change the text to something like: "You are attempting to mutate state where it is not allowed"... or something like that :)

**Comment by christianalfoni** [comment]
Hi there,

Just closing this due to inacitivity. This is the message now:

```js
        `proxy-state-tree - You are mutating the path "${path}", but it is not allowed. The following could have happened:
        
        - The mutation is explicitly being blocket
        - You are passing state to a 3rd party tool trying to manipulate the state
        - You are running asynchronous code and forgot to "await" its execution
```

**Pull request closed without merging**